utils/whatsapp/api.go

```diff
@@ -136,6 +136,16 @@ type MOPayload struct {
 			Timestamp int64 `json:"timestamp"`
 
 			OptIn *struct {
```

Comment:
@norkans7 I didn't realize whatsapp and facebook were using the same structs here... should they be? Should this package be the `meta` package instead of the `whatsapp` package?

Reply:
yes, they share the structs. we can rename that package